idaes/models/properties/activity_coeff_models/activity_coeff_prop_pack.py

```diff
@@ -2124,8 +2122,9 @@ def calculate_scaling_factors(self):
         if self.is_property_constructed("temperature_bubble"):
             iscale.set_scaling_factor(self.temperature_bubble, sf_T)
         if self.is_property_constructed("eq_temperature_bubble"):
+            sf_p = iscale.get_scaling_factor(self.pressure)
             iscale.constraint_scaling_transform(
-                self.eq_temperature_bubble, sf_T, overwrite=False
+                self.eq_temperature_bubble, sf_p, overwrite=False
```

Comment:
A comment on why P is the correct scaling to use would be useful here so that someone does not undo this in the future.

Reply:
That equation gives the bubble temperature implicitly, with the sum of partial pressures at the bubble temperature equaling the total pressure. Ideally we'd have people define scaling factors or rules to generate scaling factors immediately after they write the equation, so they don't forget what they wrote when it comes time to scale.

Reply:
My point was to leave a note to that effect so that someone doesn't think this was a typo.